core/templates/dev/head/components/state-directives/rule-editor/rule-editor.directive.ts

```diff
@@ -16,7 +16,12 @@
  * @fileoverview Directive for the rule editor.
  */
 
+require(
+  'components/state-directives/rule-editor/rule-type-selector.directive.ts');
+require('components/forms/custom-forms-directives/object-editor.directive.ts');
+require('components/forms/custom-forms-directives/html-select.directive.ts');
```

Comment:
Ordering.

Reply:
Done

Reply:
Not done. `require('components/forms/custom-forms-directives/html-select.directive.ts');` should be first.

Reply:
Done